### PR TITLE
Update CLion documentation.

### PR DESCRIPTION
**Pull request by TheJhonny007**
Updates the documentation on CLion.

### CODE REVIEW

**Comment by octfx** [comment]
Release and Debug profiles tested on Windows without errors. (MSVC 19)  
Building fails due to an unrelated error.

Linux currently untested, as my cmake verison won't find glm.

**Comment by TheJhonny007** [comment]
> Linux currently untested, as my cmake verison won't find glm.

Is it because of an issue on your end or because of the CLion config?

Can you accept the request or suggest changes?

**Comment by octfx** [comment]
> > Linux currently untested, as my cmake verison won't find glm.
> 
> Is it because of an issue on your end or because of the CLion config?
> 
> Can you accept the request or suggest changes?

My guess is, that it is an error on my end 😄
Changes look good.

File: docs/ide-setup.md [hunk 1]
```diff
@@ -17,94 +17,111 @@ Below you find some instructions on how to setup your preferred IDE for CosmoSco
...
+  - **Executable** -> _Select other_: `<path to cosmoscout>\install\windows-Release\bin\cosmoscout.exe`
+  - **Program arguments**: `--settings=../share/config/simple_desktop.json -vista vista.ini`
+  - **Environment variables**: `VISTACORELIBS_DRIVER_PLUGIN_DIRS=..\lib\DriverPlugins;PATH=..\lib\;%PATH%`
+  - **Before launch** -> __--__ -> _Build_
+  - **Before launch** -> __+__ -> _Install_
+- Debug profile
+  - **Target**: `cosmoscout`
+  - **Executable** -> _Select other_: `<path to cosmoscout>\install\windows-Debug\bin\cosmoscout.exe`
+  - **Program arguments**: `--settings=../share/config/simple_desktop.json -vista vista.ini`
+  - **Environment variables**: `VISTACORELIBS_DRIVER_PLUGIN_DIRS=..\lib\DriverPlugins;PATH=..\lib\;%PATH%`
```
Comment (octfx) [inline docs/ide-setup.md:112]: Are these env vars required? I had no issues starting comsocout without them.

File: docs/ide-setup.md [hunk 2]
```diff
@@ -17,94 +17,111 @@ Below you find some instructions on how to setup your preferred IDE for CosmoSco
...
+  - **Build options**: `--parallel <your core count>`
+  - **Environment**: If you haven't set `BOOST_ROOT` as a system wide environment variable you need to set it here!
+  - **CMake options**:
+    ```
+    -G "Visual Studio 16 2019" -A x64
+    -DCMAKE_INSTALL_PREFIX="<path to cosmoscout>\install\windows-Debug"
+    -DCOSMOSCOUT_EXTERNALS_DIR="<path to cosmoscout>\install\windows-externals-Debug"
+    ```
+
 - Go to: _Run/Debug Configuration_ -> _CMake Application_ -> _cosmoscout_
```
Comment (octfx) [inline docs/ide-setup.md:100]: Should here be an info that the exe is only there if the app has been build? Or is this implied? 😄

Comment (TheJhonny007) [reply docs/ide-setup.md:104]: I added an instruction to run the install task first.

**Pull request merged**